src/AuthError.ts

```diff
@@ -9,10 +9,13 @@ export enum AuthErrorCode {
   FAILED_TO_DECODE = 'FAILED_TO_DECODE',
   UNKNOWN_ISSUER = 'UNKNOWN_ISSUER',
   WRONG_ISSUER = 'WRONG_ISSUER',
+  WRONG_AUDIENCE = 'WRONG_AUDIENCE',
```

Comment:
Asymmetric validation algorithm includes checking `aud` from JWT is the site base URL.